source/beamline/CLS/CLSShutters.cpp

```diff
@@ -123,20 +123,36 @@ bool CLSShutters::isClosed() const
 	return result;
 }
 
+bool CLSShutters::isSafetyShutterOpen() const
+{
+	if (safetyShutter_) {
+		return safetyShutter_->isConnected() && isChildState1(safetyShutter_);
+	}
+
+	return true;
```

Comment:
I think that it is probably better to return false as a default for whether this kind of equipment is open.